docs/csharp/language-reference/language-specification/toc.md

```diff
@@ -18,3 +18,10 @@
 ## [Attributes](../../../../_csharplang/spec/attributes.md)
 ## [Unsafe code](../../../../_csharplang/spec/unsafe-code.md)
 ## [Documentation comments](../../../../_csharplang/spec/documentation-comments.md)
+# C# 7.0 language proposals 
```

Comment:
shouldn't this be on its own TOC file under the proposals folder? Also a couple of paths are not working right now such as https://review.docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-7.0/?branch=pr-en-us-11026 or https://review.docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/?branch=pr-en-us-11026

It's fine if you don't want to do that, but then I think I'd add an entry above the draft and language proposals called "C# language specification and proposals" to match what they clicked on the TOC.

Reply:
I addressed this with redirects to the first result in each folder.